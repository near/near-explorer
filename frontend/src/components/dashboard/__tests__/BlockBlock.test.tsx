import renderer from "react-test-renderer";

import BlocksBlock from "../DashboardBlocksBlock";

describe("<BlockDetails />", () => {
  it("renders", () => {
    expect(
      renderer.create(
        <BlocksBlock
          block={{
            transactionsCount: 3,
            timestamp: Number(new Date(2019, 1, 1)),
            hash: "EVvWW1S9BFaEjY1JBNSdstb7ZTtTFjQ6cygkbw1KY4tL",
            prevHash: "EVvWW1S9BFaEjY1JBNSdstb7Zjghjlyguiygfhgu",
            height: 12345,
            gasPrice: "5000",
            gasUsed: 0,
          }}
        />
      )
    ).toMatchSnapshot();
  });
});
