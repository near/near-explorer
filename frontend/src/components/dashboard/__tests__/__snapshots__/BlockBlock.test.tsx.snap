// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockDetails /> renders 1`] = `
<div
  className="col-xl-6 col-12"
>
  <a
    className="jsx-1776142549 dashboard-blocks-block-link"
    href="/blocks/EVvWW1S9BFaEjY1JBNSdstb7ZTtTFjQ6cygkbw1KY4tL"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <div
      className="dashboard-blocks-block card"
    >
      <div
        className="dashboard-blocks-block-title card-title h5"
      >
        #
        12345
      </div>
      <div
        className="dashboard-blocks-block-content card-body"
      >
        <p
          className="jsx-1776142549 dashboard-blocks-block-content-p"
        >
          <img
            className="jsx-1776142549"
            src="/static/images/icon-m-transaction.svg"
          />
          3
        </p>
        <div
          className="row no-gutters"
        >
          <div
            className="col-md-7 col-7"
          >
            <span
              className="jsx-1776142549 dashboard-blocks-block-content-p-footer"
            >
              EVvWW1
              ...
            </span>
          </div>
          <div
            className="align-self-center text-right col-md-5 col-5"
          >
            <span
              className="jsx-1776142549 dashboard-blocks-block-timer"
            >
              <span>
                1s ago
              </span>
            </span>
          </div>
        </div>
      </div>
    </div>
  </a>
</div>
`;
